addons/website/static/tests/tours/snippet_image_crop.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module */
+
+import wTourUtils from "@website/js/tours/tour_utils";
+
+wTourUtils.registerWebsitePreviewTour('snippet_image_crop', {
+    test: true,
+    url: '/',
+    edition: true,
+}, () => [
+    wTourUtils.dragNDrop({
+        id: 's_text_image',
+        name: 'Text - Image',
+    }),
+    {
+        content: "Select image",
+        trigger: 'iframe .s_text_image img',
+    },
+    {
+        content: "Replace image",
+        trigger: 'iframe .s_text_image img',
+        run: 'dblclick',
+    },
+    {
+        content: "Search for the illustration images",
+        trigger: ".o_select_media_dialog .o_we_search",
+        run: "text city",
+    },
+    {
+        content: "Select the first illustration image",
+        trigger: ".o_we_existing_attachments a.o_we_media_author:contains('Undraw')"
```

Comment:
Instead of searching with 'Undraw' string I would select first image from searched images, do we really need this step? as following step will select the image, if we keep this step then this step should be check step.

Reply:
@msh-odoo ,Thanks for noticing, initially i tested it with another selector which caused it to break the scenario as few svg images are preloaded in runbot and they are crop-able, i will update it and remove Undraw which we actually dont need when we have `a:o_we_media_author`.